Codigo/CharacterPersistence.bas

```diff
@@ -402,6 +380,14 @@ ErrorHandler:
 
 End Function
 
+Public Sub UpdateCharacterPatreonCredits(ByVal UserIndex As Integer)
+    Dim RS As ADODB.Recordset
+    With UserList(UserIndex)
+        Set RS = Query("Select offline_patron_credits from account where id = ?;", .AccountID)
```

Comment:
Either rename ot change to `UPDATE` instead of `SELECT`

We cannot write directly to `offline_patron_credits` because there are other processes doing the same so you could overwrite the value.

Reply:
we are doing a query here, why would we use update?
this updates de character with the credits in the account

besides that we do write and read to that field this change is done so we only do it when a player buy something instead of every login/logout

Reply:
Ok I see the confusion here, the name `UpdateCharacterPatreonCredits` suggests updating the DB.

Let's please rename `UpdateCharacterPatreonCredits` to 'LoadPatronCredots`.